gtda/diagrams/_metrics.py

```diff
@@ -12,6 +12,18 @@
 from ._utils import _subdiagrams, _sample_image
 
 
+def silhouettes(diagrams, sampling, order, **kwargs):
+    """Input: a batch of persistence diagrams with a 1D sampling"""
```

Comment:
`sampling` is not a 1D array, it is in fact a 3D array.

Reply:
True -i believe the point to be precise that we sample a one-dimensional space. But, indeed, the first executed line implies that the array has three axes.

Reply:
@wreise, thanks. It is great!